factory_bot_rails.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name        = "factory_bot_rails"
-  s.version     = "5.1.1"
+  s.version     = "5.1.1.1"
```

Comment:
Let's remove this change, we'll bump the version when we cut a new release.

Reply:
sure, but I have to have it while my code is not in the master.